radix-engine/src/track/interface.rs

```diff
@@ -207,6 +207,8 @@ pub enum StoreAccess {
     /// A new entry has been added to track
     /// System limits how many items that can be tracked.
     NewEntryInTrack,
+    /// Substate has been read from heap
+    ReadFromHeap,
```

Comment:
Hrm, this is the wrong place to add "Heap" related info as `Track` should not have any notion of a `Heap`